examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImReadCommand.kt

```diff
@@ -35,9 +37,86 @@ class PairOnNetworkLongImReadCommand(
       setFailure("read failure")
     }
 
+    fun checkAttributeTlv(attribute: AttributeState): Boolean = 
+      attribute.getTlv().joinToString().equals("""21, 40, 16, 24""").also {
+      if (!it) {
+          logger.log(Level.INFO, "read attribute tlv failure")
+      }
+    };
+
+    fun checkAttributeJson(attribute: AttributeState): Boolean =
+      attribute.getJson().toString().equals("""{"16:BOOL":false}""").also {
+        if (!it) {
+            logger.log(Level.INFO, "read attribute json failure")
+        }
+    };
+
+
+    fun checkEventTlv(event: EventState): Boolean =
+    event.getTlv().joinToString().equals("""21, 53, 0, 36, 0, 1, 24, 24""").also {
+      if (!it) {
+          logger.log(Level.INFO, "read event tlv failure")
+      }
+    };
+
+    fun checkEventJson(event: EventState): Boolean =
+      event.getJson().toString().equals("""{"0:STRUCT":{"0:UINT":1}}""").also {
+        if (!it) {
+            logger.log(Level.INFO, "read event json failure")
+        }
+    };
+
+    fun checkAllAttributeForBasicCluster(cluster: String): Boolean {
+      val expected =
+        """{"16:BOOL":false,""" +
+          """"65531:ARRAY-UINT":[""" +
+          """0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,18,19,20,65528,65529,65531,65532,65533]}"""
+      if (!cluster.equals(expected)) {
+        logger.log(Level.INFO, "read all attributes failure from basic cluster")
+        return false
+      }
+      return true
+    }
+
     override fun onReport(nodeState: NodeState) {
-      logger.log(Level.INFO, "Read receve onReport")
-      setSuccess()
+      logger.log(Level.INFO, nodeState.toString())
+      var successAttribute: Boolean = true
+      var successEvent: Boolean = true
+      var successCluster: Boolean = true
+      val attribute =
+        nodeState
+          .getEndpointState(0)
+          ?.getClusterState(CLUSTER_ID_BASIC)
+          ?.getAttributeState(ATTR_ID_LOCAL_CONFIG_DISABLED)
+
+      val event =
+        nodeState
+          .getEndpointState(0)
+          ?.getClusterState(CLUSTER_ID_BASIC)
+          ?.getEventState(EVENT_ID_START_UP)
+
+      val clusterAttributes =
+        nodeState.getEndpointState(0)?.getClusterState(CLUSTER_ID_BASIC)?.getAttributesJson()
+
+      if (attribute != null) {
+        successAttribute = checkAttributeTlv(attribute) && checkAttributeJson(attribute)
+      }
+
+      if (event != null) {
```

Comment:
You probably also want to check if the `event` is empty? Also probably want to name it to indicate it's a collection:

```
val events = ...
if (events != null && events.isNotEmpty())
```

Reply:
Done, thanks

Reply:
thank you!